integration-tests/opentelemetry/src/test/java/io/quarkus/it/opentelemetry/OpenTelemetryTestCase.java

```diff
@@ -189,7 +189,7 @@ void testEmptyClientPath() {
                 Assertions.assertEquals("/", spanData.get("attr_http.target"));
                 Assertions.assertEquals(pathParamUrl.getAuthority(), spanData.get("attr_http.host"));
                 Assertions.assertEquals("http", spanData.get("attr_http.scheme"));
-                Assertions.assertNull(spanData.get("attr_http.route"));
```

Comment:
I'm not sure the `route` should be `/` given it represents a path template.

If there is no path, it should probably be null/empty

Reply:
Yes, the spec is not very clear about this. Let me try to clarify this with the OTel group. In the meanwhile, I'll revert this.